src/System.Text.Json/tests/Serialization/ExceptionTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public static partial class ExceptionTests
+    {
+        [Fact]
+        public static void RootException()
+        {
+            bool exceptionThrown = false;
+            try
+            {
+                int i2 = JsonSerializer.Parse<int>("12bad");
```

Comment:
nit: an alternative approach (without the `exceptionThrown` flag) would be to add right after the method call.
```C#
Assert.True(false, "Some useful message").
```

For example:
https://github.com/dotnet/corefx/blob/830bf7fc0346d9453d85ad4ec726c92ac3e72998/src/System.Text.Json/tests/Utf8JsonReaderTests.cs#L865

Reply:
I like that approach - I'll update.